### PR TITLE
14.0 mig sale commission pricelist

### PR DESCRIPTION
**Pull request by SilvioGregorini**

### CODE REVIEW

**Review by matteoopenf — APPROVED** [review]

**Comment by anajuaristi** [comment]
Functionally tested LGFM. :+1:

**Comment by matteoopenf** [comment]
@OCA/community-maintainers  is possibile merge?

File: sale_commission_pricelist/__manifest__.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 {
     "name": "Sales commissions by pricelist",
-    "version": "13.0.1.1.0",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Tecnativa, Odoo Community Association (OCA), Openforce Srls",
```
Comment (pedrobaeza) [inline sale_commission_pricelist/__manifest__.py]: You can't add yourself as author of the module for such simple module migration. Put yourself as contributor.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: sale_commission_pricelist/__manifest__.py (hunk 1 above)
Comment (matteoopenf) [reply sale_commission_pricelist/__manifest__.py]: but there is Tecnativa is normal?

Comment (pedrobaeza) [reply sale_commission_pricelist/__manifest__.py]: Of course. We created the module.

Comment (SilvioGregorini) [reply sale_commission_pricelist/__manifest__.py]: @pedrobaeza Done

**Comment by pedrobaeza** [comment]
Please squash migration commits and never put merge commits in a pull request.

**Pull request closed without merging**